apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/os/OSUtil.java

```diff
@@ -116,6 +116,9 @@ public static int getProcessNo() {
         }
         List<String> allIPV4 = getAllIPV4();
         if (allIPV4.size() > 0) {
+            if (allIPV4.size() >= ipv4Size) {
```

Comment:
if `allIPV4.size() == ipv4Size`, there is no need to `subList`, it must be itself 

```suggestion
            if (allIPV4.size() > ipv4Size) {
```